.travis.yml

```diff
@@ -146,17 +146,13 @@ jobs:
   - <<: *stage_deploy
     if: tag IS present
     <<: *python_3_7_mixture
-    install: skip
-    script: skip
-    deploy:
-      provider: pypi
-      on:
-        all_branches: true
-      user: jaraco
-      password:
-        secure: RAfz06AINvz7bfij/YhfkAreRqamgxS8a6jSRNxntYhtJke3ZszUbIDag8+n1I+G5XT2LnMhHqPNR7Plc+AeMz7VXTuy+b81Li5kse20NYlPhd7mBVmTUpXtqYQashV5J39F4qkATBLznCOrMEomM07VTXjO/o2hmQuXniab2Uo=
-      distributions: dists
-      skip_cleanup: true
+    before_script: skip
+    env:
+    - TWINE_USERNAME=jaraco
+    # TWINE_PASSWORD
+    - secure: ahqqwT5dPGZMyOr2NyavsSQAaWgE/RKy0bD0bT55MdXfVh7lpXU6sVupxKJT6/uwS8uaAahvxBjr6mTvTs7pHuUPJ0MVsFb+LnQVUzlqc9Z47tE7nWGn/mYXb+mxN/VBhlUX/Y4rQMxcW6VFqOMSQBPQ2EukIh76nerbvQC1ypA=
+    - TOX_TESTENV_PASSENV="TWINE_USERNAME TWINE_PASSWORD"
+    script: tox -e release
```

Comment:
Curious whether we'll be able to use Actions for this once events other than `push` will be available... https://github.com/cherrypy/cheroot/actions

Reply:
I'm not familiar with actions.

Reply:
It's a brand-new toy: https://github.com/features/actions. We have partial access to them. It looks like we can build container-based pipelines on top of this reacting to various GitHub events. And there's some pre-cooked building blocks which is neat.